packages/legacy-client/src/ontology-runtime/ontologyProvider/calls/getObject.ts

```diff
@@ -14,32 +14,33 @@
  * limitations under the License.
  */
 
+import type { ThinClient } from "@osdk/api";
 import { createOpenApiRequest } from "@osdk/api";
 import { getObjectV2 } from "@osdk/gateway/requests";
+import { convertWireToOsdkObject } from "../../../client/convertWireToOsdkObject";
 import type { OntologyObject } from "../../baseTypes";
 import { GetObjectErrorHandler, handleGetObjectError } from "../ErrorHandlers";
 import type { GetObjectError } from "../Errors";
 import type { Result } from "../Result";
-import type { ClientContext } from "./ClientContext";
 import { wrapResult } from "./util/wrapResult";
 
 export async function getObject<T extends OntologyObject>(
-  context: ClientContext,
+  client: ThinClient<any>,
```

Comment:
I can take care of this in a follow up but we can probably change all the calls to take the OntologyDefinition as a type and use that here